### DISCUSSION TITLE
Object tags not sync in site replicate

### DISCUSSION THREAD

**Discussion by umjsio** [minio/minio#18643] [Q&A] [answered]
- RELEASE.2023-12-09T18-17-51Z
- site replicate is set up between two clusters. 

Everything is working fine, however, when I try to update the tag of an existing object, the changes are not getting synced to the other site. 
May I know if this is expected behavior? 
Thank you.

**Comment by harshavardhana** [comment]
@umjsio do you have `mc admin trace -v` output when you update the tags ? For both clusters please.

**Reply by umjsio** [reply]
Thank you for your response, I updated an object's tag at site B, please check the attached files.
[siteb.txt](https://github.com/minio/minio/files/13657996/siteb.txt)
[sitea.txt](https://github.com/minio/minio/files/13657997/sitea.txt)

**Reply by harshavardhana** [reply, marked as answer]
This was a bug fixed in latest release
